source/game/field/KCollisionTypes.hh

```diff
@@ -78,12 +81,24 @@ typedef enum {
             KCL_TYPE_BIT(COL_TYPE_STICKY_ROAD) | KCL_TYPE_BIT(COL_TYPE_ROAD2) | \
             KCL_TYPE_BIT(COL_TYPE_ROTATING_ROAD))
 
+/*D010F000*/
 #define KCL_TYPE_WALL \
     (KCL_TYPE_BIT(COL_TYPE_WALL) | KCL_TYPE_BIT(COL_TYPE_INVISIBLE_WALL) | \
             KCL_TYPE_BIT(COL_TYPE_ITEM_WALL) | KCL_TYPE_BIT(COL_TYPE_WALL_2) | \
             KCL_TYPE_BIT(COL_TYPE_PLAYER_WALL) | KCL_TYPE_BIT(COL_TYPE_HALFPIPE_INVISIBLE_WALL) | \
             KCL_TYPE_BIT(COL_TYPE_SPECIAL_WALL) | KCL_TYPE_BIT(COL_TYPE_INVISIBLE_WALL2))
 
+/*EFFFBDFF*/
+#define KCL_TYPE_VEHICLE_COLLIDEABLE \
+    (~KCL_TYPE_BIT(COL_TYPE_ITEM_ROAD) & ~KCL_TYPE_BIT(COL_TYPE_ITEM_WALL) & \
+            ~KCL_TYPE_BIT(COL_TYPE_HALFPIPE_INVISIBLE_WALL))
+
+/*EAF8BDFF*/
+#define KCL_TYPE_IDK \
+    ((KCL_TYPE_FLOOR | KCL_TYPE_WALL | KCL_TYPE_BIT(COL_TYPE_WEAK_WALL) | \
+             KCL_TYPE_BIT(COL_TYPE_ITEM_STATE_MODIFIER)) & \
+            KCL_TYPE_VEHICLE_COLLIDEABLE)
```

Comment:
We'll need to give this a proper name

Reply:
This is also equivalent to the following expression:
```
#define KCL_TYPE_IDK \
    (KCL_TYPE_VEHICLE_COLLIDEABLE & ~KCL_TYPE_BIT(COL_TYPE_SOUND_TRIGGER) & \
            ~KCL_TYPE_BIT(COL_TYPE_EFFECT_TRIGGER) & ~KCL_TYPE_BIT(COL_TYPE_FALL_BOUNDARY) & \
            ~KCL_TYPE_BIT(COL_TYPE_CANNON_TRIGGER) & \
            ~KCL_TYPE_BIT(COL_TYPE_FORCE_RECALCULATE_ROUTE))
```
I almost think *this* is more readable. It makes it clear that this is probably "visible" collision (cannon/sound/effect triggers are invisible, as is the oob fall boundary)